test/integration/packwerk/custom_executable_integration_test.rb

```diff
@@ -128,6 +128,33 @@ class CustomExecutableIntegrationTest < Minitest::Test
       end
     end
 
+    test "'packwerk check' does not blow up when parsing files with syntax issues from a false positive association" do
+      open_app_file(TIMELINE_PATH.join("app", "models", "bad_file.rb")) do |file|
+        # This is an example of a file that references `belongs_to`, but as an ActiveAdmin API
+        # rather than as a Rails Association concept
+        content = <<~CONTENT
+          require 'rails_helper'
+
+          RSpec.describe ActiveAdmin::Resource::BelongsTo do
+            it "should have an owner" do
+              expect(belongs_to.owner).to eq post_config
```

Comment:
I'm not sure I get this example. It is failing because `belongs_to` doesn't exist? Is packwerk executing the Ruby code instead of just parsing it?